webapp/accounts/templates/registration/login.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'geek_text/base.html' %}
```

Comment:
This template `login.html` should be namespaced under the `accounts` app, not the `registration` namespace.
Also, why is there a new login page here? What happened to the old one done in the previous Pull Request?

Reply:
I will fix that